spring-social-core/src/main/java/org/springframework/social/connect/redis/data/SocialRedisConnection.java

```diff
@@ -0,0 +1,126 @@
+package org.springframework.social.connect.redis.data;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.redis.core.RedisHash;
+import org.springframework.data.redis.core.index.Indexed;
+
+@RedisHash
```

Comment:
Pinning all elements of `SocialRedisConnection` to a specific slot for potential clustered Redis usage might be a good idea. This would allow all finder methods to operate directly on the server - see: [7.8. Redis Repositories running on Cluster](http://docs.spring.io/spring-data/redis/docs/current/reference/html/#redis.repositories.cluster).